.gitignore

```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+package-lock.json
```

Comment:
we should not ignore `package-lock.json` as it helps maintain consistency between dev and prod systems.

Reply:
![image](https://user-images.githubusercontent.com/46242073/107077833-6ef2b800-6813-11eb-9b20-395366efa611.png)

you told to do this na?

Reply:
i meant just delete the file. Not add it in git ignore.